openfl-docker/licenses.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Downloads thirdparty licenses.
+# Save the list of installed packages to base_packages.txt
+dpkg --get-selections | grep -v deinstall | awk '{print $1}' > base_packages.txt
+
+# Clean up APT lists
+rm -rf /var/lib/apt/lists/*
+
+# Update the package list and install specified packages
+apt-get update
+apt-get install -y --no-install-recommends --fix-missing \
+    openssh-server=* \
+    curl=* \
+    ca-certificates=*
```

Comment:
These installs need to go in dockerfile as they aren't related to licenses?

Reply:
Removed.